src/utils/transaction.ts

```diff
@@ -314,6 +421,308 @@ export const buildCrossChainAction = async (
     }
   }
 
+  if (transactionBlock.type === TRANSACTION_BLOCK_TYPE.PLR_DAO_STAKE
+    && !!transactionBlock?.values?.fromChainId
+    && !!transactionBlock?.values?.fromAsset
```

Comment:
please use ONE pattern how you do multi line logic operators, in one piece of code you put it at back of line, here at front, at front is preferred, thank you
```suggestion
    && !!transactionBlock?.values?.fromChainId
    && !!transactionBlock?.values?.fromAsset
```